client/src/styles/legacy/_forms.scss

```diff
@@ -13,11 +13,6 @@
 // need this for consistent <fieldset> behaviour
 fieldset {
   width: 100%;
-
-  // fix firefox fieldset width behaviour http://stackoverflow.com/a/17863685
-  @-moz-document url-prefix() {
-    display: table-cell;
-  }
```

Comment:
Had to manually fix this linting problem:
```text
client/src/styles/legacy/_forms.scss
  18:3  ✖  Unexpected vendor-prefixed at-rule "@-moz-document"  at-rule-no-vendor-prefix
```
As per the edit to the linked stackoverflow answer:
> The [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=504622#c5) described below is fixed as of Firefox 53